keyboards/viktus/tx_roundup_pad/info.json

```diff
@@ -0,0 +1,53 @@
+{
+    "manufacturer": "Viktus Design LLC",
+    "keyboard_name": "TX Roundup Pad",
+    "maintainer": "BlindAssassin111",
+    "build": {
+        "lto": true
+    },
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "matrix_pins": {
+        "cols": ["F7", "F6", "F5", "F4"],
+        "rows": ["B5", "B4", "E6", "D7", "C6", "D4"]
+    },
+    "url": "https://viktus.design",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x5458",
+        "vid": "0x5644"
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                {"label": "K00", "matrix": [0, 0], "x": 0, "y": 0},
+                {"label": "K01", "matrix": [0, 1], "x": 1, "y": 0},
+                {"label": "K02", "matrix": [0, 2], "x": 2, "y": 0},
+                {"label": "K03", "matrix": [0, 3], "x": 3, "y": 0},
+                {"label": "K10", "matrix": [1, 0], "x": 0, "y": 1},
+                {"label": "K11", "matrix": [1, 1], "x": 1, "y": 1},
+                {"label": "K12", "matrix": [1, 2], "x": 2, "y": 1},
+                {"label": "K13", "matrix": [1, 3], "x": 3, "y": 1},
+                {"label": "K20", "matrix": [2, 0], "x": 0, "y": 2},
+                {"label": "K21", "matrix": [2, 1], "x": 1, "y": 2},
+                {"label": "K22", "matrix": [2, 2], "x": 2, "y": 2},
+                {"label": "K23", "matrix": [2, 3], "x": 3, "y": 2, "h": 2},
+                {"label": "K30", "matrix": [3, 0], "x": 0, "y": 3},
+                {"label": "K31", "matrix": [3, 1], "x": 1, "y": 3},
+                {"label": "K32", "matrix": [3, 2], "x": 2, "y": 3},
+                {"label": "K40", "matrix": [4, 0], "x": 0, "y": 4},
```

Comment:
```suggestion
                {"label": "K30", "matrix": [3, 0], "x": 0, "y": 3},
                {"label": "K31", "matrix": [3, 1], "x": 1, "y": 3},
                {"label": "K32", "matrix": [3, 2], "x": 2, "y": 3},
                {"label": "K23", "matrix": [2, 3], "x": 3, "y": 2, "h": 2},
                {"label": "K40", "matrix": [4, 0], "x": 0, "y": 4},
```

Reply:
The rest made sense, but I am unsure what the reasoning for the reorder here is. Unsure if I messed something up and just not seeing it or what specifically you saw.

Reply:
The reordering is so that it fits the community layout properly.  The order matters for that.  Reference here: 
https://github.com/qmk/qmk_firmware/blob/master/layouts/default/numpad_6x4/info.json